client-js/app/elements/app-link.ts

```diff
@@ -0,0 +1,30 @@
+const MOUSE_MAIN_BUTTON = 0;
+
+@component('app-link')
+@extend('a')
+export class AppLink extends polymer.Base {
+
+  @property({type: Array})
+  path: string[];
+
+  isSpecialClick(e): boolean {
+    return (e.ctrlKey || e.metaKey || e.button !== MOUSE_MAIN_BUTTON);
+  }
+
+  @listen('click')
+  onClick(e) {
+    if (this.isSpecialClick(e)) return;
```

Comment:
The js style guide points to the c++ style guide which allows single line conditionals (https://google.github.io/styleguide/cppguide.html#Conditionals), so I went with that. I agree that everyone is different on this; we should set up automated linting/formatting and just use that :-)